### PR TITLE
Adding PageImages Namespaces to my Wiki

### PR DESCRIPTION
**Pull request by Njsmooth**
Added namespaces for PageImages to display images in page previews for more namespaces in my wiki. The setting wasn't already there, so I tried to add it in. I may have done it incorrectly, though...

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -3436,6 +3436,12 @@
 		'default' => true,
 	],
 
+	// Page Images
+	'wgPageImagesNamespaces' = [
+		'default' => NS_MAIN,
```
Comment (Universal-Omega) [inline LocalSettings.php:3441]: This should be an array.

File: LocalSettings.php [hunk 2]
```diff
@@ -3436,6 +3436,12 @@
...
+		'vgportdbwiki' => [NS_MAIN, NS_PLATFORM, NS_SERIES, NS_COMPANY],
```
Comment (Universal-Omega) [inline LocalSettings.php:3442]: This should be reformatted so each constant is on a new line. Also, as for custom namespaces, I don't think the constants will work and need to be ID directly though not positive.

**Review by Universal-Omega — DISMISSED** [review]

**Comment by Universal-Omega** [comment]
Also syntax error.

File: LocalSettings.php [hunk 3]
```diff
@@ -3436,6 +3436,12 @@
 		'default' => true,
 	],
 
+	// Page Images
+	'wgPageImagesNamespaces' = [
```
Comment (The-Voidwalker) [inline LocalSettings.php:3440]:
```suggestion
	'wgPageImagesNamespaces' => [
```

**Review by The-Voidwalker — CHANGES_REQUESTED** [review]

File: LocalSettings.php (hunk 2 above)
Comment (Njsmooth) [reply LocalSettings.php:3442]: I'm not even entirely sure what the constants are (if they even have one that is). Do you know where I'd be able to check?

Comment (Universal-Omega) [reply LocalSettings.php:3442]: > I'm not even entirely sure what the constants are (if they even have one that is). Do you know where I'd be able to check?

If you see Special:ManageWiki/namespaces and select your namespace and select edit, the final subpage in the address bar is the namespace ID. (Custom ones are always above 3000)

Comment (Njsmooth) [reply LocalSettings.php:3442]: Oh, I know about the ID, but I wasn't sure about the constant. Also, id I were to try using the ID's, how would I type that in the code?

Comment (Universal-Omega) [reply LocalSettings.php:3442]: Just replace the constants with the ID integer.

**Comment by Universal-Omega** [comment]
It should look like:
```
	// Page Images
	'wgPageImagesNamespaces' => [
		'default' => [
			NS_MAIN,
		],
		'vgportdbwiki' => [
			NS_MAIN,
			3000,
			3004,
			3006,
		],
	],
```

File: LocalSettings.php (hunk 1 above)
Comment (Universal-Omega) [reply LocalSettings.php:3441]: I'll merge this once this is done.

Comment (Njsmooth) [reply LocalSettings.php]: Did I fix it?

**Pull request merged**